targets/targets.json

```diff
@@ -630,6 +630,19 @@
         "release_versions": ["2", "5"],
         "device_name": "MK82FN256xxx15"
     },
+    "LPC54114": {
```

Comment:
wouldnt be better to name this ``LPCXpresso54114``? and having lpc54114 for mcu (MCU_name) ?